README.markdown

```diff
@@ -22,6 +22,7 @@ data-info.max-paths-in-request       = 1000
 data-info.metadata.base-url          = http://example.org:31331
 data-info.perms-filter               = rodsadmin_acl,rodsBoot,rodsadmin,admin_proxy
 data-info.port                       = 60000
+data-info.commons.base               = http://datacommons.example.org/
```

Comment:
Since we're migrating to having configs stored in Kubernetes, I updated the example in the README, but I did not update the configuration template. The templates used to generate the configs stored in Kubernetes still need to be updated.